experimental/packages/opentelemetry-instrumentation/src/platform/node/RequireInTheMiddleSingleton.ts

```diff
@@ -59,9 +59,15 @@ export class RequireInTheMiddleSingleton {
   }
 
   private _initialize() {
+    // A list of strings separated by comma e.g. fastify,fs
+    const modules = process.env.PATCH_ONLY
```

Comment:
@Flarna 

This is a simple solution to the referenced ticket. It is also possible to add a config option for the `opentelemetry.NodeSDK`, but for my use case I am happy with an environment variable because I am not using the SDK.

If you accept this solution, I will add a test and the changelog entry.
Feel free to suggest a different env variable name.

Reply:
I think the name of the env variable should start with `OTEL_` (or maybe `_OTEL` or `OTEL_JS` as not from spec).